### PR TITLE
JavaScript Best practices project

### PR DESCRIPTION
**Pull request by KingKowa**
- This project is about conforming to best practices in JavaScript code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    li.appendChild(garbage);
+    li.appendChild(span);
+    list.appendChild(li);
+  });
+};
+
+populateData();
+
+document.addEventListener('click', (event) => {
+  if (event.target.classList.contains('checkbox')) {
```
Comment (maury18) [inline src/index.js:44]: Kindly, try to add all the if statement in just one so you don't have to clone the code

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  if (event.target.classList.contains('garbage')) {
+    todo.removeTodo(Number(event.target.classList[0]));
+    localStorage.setItem('todo', JSON.stringify(todo.getTodos()));
+    populateData();
+  }
+
+  if (event.target.classList.contains('clear-button')) {
+    todo.clearCompleted();
+    localStorage.setItem('todo', JSON.stringify(todo.getTodos()));
```
Comment (maury18) [inline src/index.js:63]: also here, instead of cloning it would be better to reuse it as a function

**Review by maury18 — COMMENTED** [review]
Great job, the project is completed, there are a few optional suggestions in the comments below but nothing that cant stop it from blocking your project as completed

File: src/index.js (hunk 1 above)
Comment (KingKowa) [reply src/index.js:44]: Thank you for your feedback

File: src/index.js (hunk 2 above)
Comment (KingKowa) [reply src/index.js:63]: Thanks for the feedback

**Pull request merged**